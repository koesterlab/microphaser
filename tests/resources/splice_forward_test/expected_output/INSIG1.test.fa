>9b8a64afbe863eeF
TATACATCCCCAGATTTCCTCTATATT
>7d13051bf18eafaF
TATACATtCCCAGATTTCCTCTATATT
>386919e3bf61c51F
TATAATGGTGTCTATCAGTATACATCC
>1de0847cb46470fF
TATAATGGTGTCTATCAGTATACATtC
>b6c07f346fd088eF
AATGGTGTCTATCAGTATACATCCCCA
>7e1436f9b762534F
AATGGTGTCTATCAGTATACATtCCCA
>907d0ed1e247d26F
GGTGTCTATCAGTATACATCCCCAGAT
>67c74a8fdcf576dF
GGTGTCTATCAGTATACATtCCCAGAT
>116ca40cee8c3d2F
GTCTATCAGTATACATCCCCAGATTTC
>74a69843f2ddbc2F
GTCTATCAGTATACATtCCCAGATTTC
>3c45a15465e5555F
TATCAGTATACATCCCCAGATTTCCTC
>b4e7467944eef77F
TATCAGTATACATtCCCAGATTTCCTC
>768e182a7bc248eF
ACATCCCCAGATTTCCTCTATATTCGT
>12f7bb5c80b6b97F
ACATtCCCAGATTTCCTCTATATTCGT
>300d1bee17b68daF
TCCCCAGATTTCCTCTATATTCGTTCT
>bcbedf057fed989F
TtCCCAGATTTCCTCTATATTCGTTCT
>159f5d84d2b5f3bF
TATACATCCCCAGATTTCCTCTATATT
>242ea7e9cdd0cfcF
TATACATtCCCAGATTTCCTCTATATT
>d2274a95f4fa1cbF
TATAATGGTGTCTATCAGTATACATCC
>5fb7492851fd200F
TATAATGGTGTCTATCAGTATACATtC
>9bc9687190cc22dF
AATGGTGTCTATCAGTATACATCCCCA
>5324ede7d387db2F
AATGGTGTCTATCAGTATACATtCCCA
>5e7f6a1964d92eaF
GGTGTCTATCAGTATACATCCCCAGAT
>ddc5b6fa97acde3F
GGTGTCTATCAGTATACATtCCCAGAT
>eb96baa4ae634ddF
GTCTATCAGTATACATCCCCAGATTTC
>88b5bad5736ef06F
GTCTATCAGTATACATtCCCAGATTTC
>7ddd96d60ecf94dF
TATCAGTATACATCCCCAGATTTCCTC
>1f43098fdb1100aF
TATCAGTATACATtCCCAGATTTCCTC
>3c9b00543aecdbcF
ACATCCCCAGATTTCCTCTATATTCGT
>52764b3302ff8ebF
ACATtCCCAGATTTCCTCTATATTCGT
>9694b5023e1403eF
TCCCCAGATTTCCTCTATATTCGTTCT
>1cf9c802b1ab876F
TtCCCAGATTTCCTCTATATTCGTTCT
>9aafe45e70a9eabF
ATACATCCCCAGATTTCCTCTATATTC
>67ba8a766cd9b87F
ATACATtCCCAGATTTCCTCTATATTC
>ce264525c05065bF
CATCCCCAGATTTCCTCTATATTCGTT
>7591c4d1b043ceeF
CATtCCCAGATTTCCTCTATATTCGTT
>e11f1b9ad93a83bF
CCCCAGATTTCCTCTATATTCGTTCTT
>7a089f37cb68ad3F
tCCCAGATTTCCTCTATATTCGTTCTT
>360090351c10833F
GTATACATCCCCAGATTTCCTCTATAT
>8610d053bc3684eF
GTATACATtCCCAGATTTCCTCTATAT
>340eecab09439a9F
GTATAATGGTGTCTATCAGTATACATC
>84b7d1aa8d77d4eF
GTATAATGGTGTCTATCAGTATACATt
>23cbe4a2dea2218F
TAATGGTGTCTATCAGTATACATCCCC
>c63a6e12310d371F
TAATGGTGTCTATCAGTATACATtCCC
>f68934dde04e51cF
TGGTGTCTATCAGTATACATCCCCAGA
>8517ce153d137c8F
TGGTGTCTATCAGTATACATtCCCAGA
>783b4f69a64adc0F
TGTCTATCAGTATACATCCCCAGATTT
>13dd6e5a771f187F
TGTCTATCAGTATACATtCCCAGATTT
>248617c4ba35425F
CTATCAGTATACATCCCCAGATTTCCT
>e315daa96f56f23F
CTATCAGTATACATtCCCAGATTTCCT
>c76f20fd1ff01f0F
TACATCCCCAGATTTCCTCTATATTCG
>345e7b215d2abeeF
TACATtCCCAGATTTCCTCTATATTCG
>0b754d64ce7ff0bF
ATCCCCAGATTTCCTCTATATTCGTTC
>ce6b9e1cc8a53bdF
ATtCCCAGATTTCCTCTATATTCGTTC
