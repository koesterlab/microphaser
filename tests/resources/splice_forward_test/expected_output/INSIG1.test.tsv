id	transcript	gene_id	gene_name	chrom	offset	freq	nvar	nsomatic	nvariant_sites	strand	positions	aa_change
9b8a64afbe863eeF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302747	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
7d13051bf18eafaF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302747	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
386919e3bf61c51F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302400	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
1de0847cb46470fF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302400	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
b6c07f346fd088eF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302403	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
7e1436f9b762534F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302403	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
907d0ed1e247d26F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302406	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
67c74a8fdcf576dF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302406	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
116ca40cee8c3d2F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302409	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
74a69843f2ddbc2F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302409	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
3c45a15465e5555F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302412	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
b4e7467944eef77F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302412	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
768e182a7bc248eF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302750	0.3055555555555556	0	0	1	Forward	155302754	p.Ser238Phe
12f7bb5c80b6b97F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302750	0.6944444444444444	1	1	1	Forward	155302754	p.Ser238Phe
300d1bee17b68daF	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302753	0.2926829268292683	0	0	1	Forward	155302754	p.Ser238Phe
bcbedf057fed989F	ENST00000340368	ENSG00000186480	INSIG1	chr7	155302753	0.7073170731707317	1	1	1	Forward	155302754	p.Ser238Phe
159f5d84d2b5f3bF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302747	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
242ea7e9cdd0cfcF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302747	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
d2274a95f4fa1cbF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302400	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
5fb7492851fd200F	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302400	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
9bc9687190cc22dF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302403	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
5324ede7d387db2F	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302403	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
5e7f6a1964d92eaF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302406	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
ddc5b6fa97acde3F	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302406	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
eb96baa4ae634ddF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302409	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
88b5bad5736ef06F	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302409	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
7ddd96d60ecf94dF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302412	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
1f43098fdb1100aF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302412	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
3c9b00543aecdbcF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302750	0.3055555555555556	0	0	1	Forward	155302754	p.Ser238Phe
52764b3302ff8ebF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302750	0.6944444444444444	1	1	1	Forward	155302754	p.Ser238Phe
9694b5023e1403eF	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302753	0.2926829268292683	0	0	1	Forward	155302754	p.Ser238Phe
1cf9c802b1ab876F	ENST00000344756	ENSG00000186480	INSIG1	chr7	155302753	0.7073170731707317	1	1	1	Forward	155302754	p.Ser238Phe
9aafe45e70a9eabF	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302748	0.35294117647058826	0	0	1	Forward	155302754	p.Ser238Phe
67ba8a766cd9b87F	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302748	0.6470588235294118	1	1	1	Forward	155302754	p.Ser238Phe
ce264525c05065bF	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302751	0.28205128205128205	0	0	1	Forward	155302754	p.Ser238Phe
7591c4d1b043ceeF	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302751	0.717948717948718	1	1	1	Forward	155302754	p.Ser238Phe
e11f1b9ad93a83bF	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302754	0.30952380952380953	0	0	1	Forward	155302754	p.Ser238Phe
7a089f37cb68ad3F	ENST00000342407	ENSG00000186480	INSIG1	chr7	155302754	0.6904761904761905	1	1	1	Forward	155302754	p.Ser238Phe
360090351c10833F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302746	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
8610d053bc3684eF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302746	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
340eecab09439a9F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302399	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
84b7d1aa8d77d4eF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302399	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
23cbe4a2dea2218F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302402	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
c63a6e12310d371F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302402	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
f68934dde04e51cF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302405	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
8517ce153d137c8F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302405	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
783b4f69a64adc0F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302408	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
13dd6e5a771f187F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302408	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
248617c4ba35425F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302411	0.36363636363636365	0	0	1	Forward	155302754	p.Ser238Phe
e315daa96f56f23F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302411	0.6363636363636364	1	1	1	Forward	155302754	p.Ser238Phe
c76f20fd1ff01f0F	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302749	0.3333333333333333	0	0	1	Forward	155302754	p.Ser238Phe
345e7b215d2abeeF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302749	0.6666666666666666	1	1	1	Forward	155302754	p.Ser238Phe
0b754d64ce7ff0bF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302752	0.2682926829268293	0	0	1	Forward	155302754	p.Ser238Phe
ce6b9e1cc8a53bdF	ENST00000476756	ENSG00000186480	INSIG1	chr7	155302752	0.7317073170731707	1	1	1	Forward	155302754	p.Ser238Phe
